index.html

```diff
@@ -0,0 +1,222 @@
+            <ul>
+                <li class="logo"><a href="#"><b>Basir Elham</b></a></li>
+                <li class="collapse-button"><a href="#"><span class="fa fa-list"></span></a></li>
+                <li class="menu">
+                    <span>
+                        <ul>
+                            <li><a href="#"><b>Portfolio</b></a></li>
+                            <li><a href="#"><b>About</b></a></li>
+                            <li><a href="#"><b>Contact</b></a></li>
+                        </ul>
+                    </span>
+                </li>
+            </ul>
+        </nav>
+        <div class="headline">
+            <div class="content">
+                <h1>Hey There <br> I'm Basir Elham</h1>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Rem saepe incidunt ut tempore ipsam cumque
+                    eveniet voluptate, eligendi cupiditate recusandae facere nam tempora quaerat maiores voluptatum
+                    quisquam minus atque unde.
+                </p>
+                <b>LET'S CONNECT</b>
+                <div class="social-media">
+                    <ul>
+                        <li><a href=""><span class="fa fa-twitter"></span></a></li>
+                        <li><a href=""><span class="fa fa-linkedin-square"></span></a></li>
+                        <li><a href=""><span class="fa fa-github-square"></span></a></li>
+                        <li><a href=""><span class="fa fa-facebook"></span></a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <section class="projects">
+            <div class="card-works">
+                <div class="image-cover">
+                    <img src="./assets/images/Snapshoot Portfolio.png" alt="">
+                </div>
+                <div class="project-info">
+                    <h1>Tonic</h1>
+                    <ul class="label">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>I'am a Software Developer
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti et id alias cumque ducimus
+                        quasi</p>
+                    <ul class="tags">
+                        <li><a href="">html</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">ruby</a></li>
+                    </ul>
+                    <p><button>See Projects</button></p>
+                </div>
+            </div>
+
+            <div class="card-works">
+                <div class="image-cover">
+                    <img src="./assets/images/Snapshoot Portfolio.png" alt="">
+                </div>
+                <div class="project-info">
+                    <h1>Tonic</h1>
+                    <ul class="label">
+                        <li>CANOPY</li>
```

Comment:
Kindly remove the remaining section, for it is not part of the requirement at this stage. 😉